lib/Parse/Lexer.cpp

```diff
@@ -1898,13 +1899,13 @@ void Lexer::lexStringLiteral(unsigned CustomDelimiterLen) {
         diagnose(--TmpPtr, diag::opening_paren);
 
         wasErroneous = true;
-        return formToken(tok::unknown, TokStart);
+        continue;
       } else {
         // As a fallback, just emit an unterminated string error.
         diagnose(CurPtr, diag::lex_unterminated_string);
         wasErroneous = true;
```

Comment:
`wasErroneous = true;` is extra too.

Reply:
How about now? All that’s gotta be left is just tests right?

Reply:
You removed the wrong one. Give me sec and I will put up a suggestion.